OpenCL-enabled PMKID attack fails to run

I installed JtR from this repo on the lastest Kali. When running the PMKID attack, I get:
```
$  ./run/john -mask=?d?d?d?d?d?d?d?d --dev=gpu --format=wpapsk-pmk-opencl pmkid-test
Device 1: Baffin [Radeon RX 560 Series]
Using default input encoding: UTF-8
Loaded 87 password hashes with 87 different salts (wpapsk-pmk-opencl, WPA/WPA2/PMF/PMKID master key [MD5/SHA-1/SHA-2 OpenCL])
Cost 1 (key version [0:PMKID 1:WPA 2:WPA2 3:802.11w]) is 0 for all loaded hashes
Options used: -I /home/user/JohnTheRipper/run/kernels -cl-mad-enable -D__GPU__ -DDEVICE_INFO=522 -DSIZEOF_SIZE_T=8 -DDEV_VER_MAJOR=2633 -DDEV_VER_MINOR=3 -D_OPENCL_COMPILER -DPLAINTEXT_LENGTH=64 -DV_WIDTH=1 -DWPAPMK $JOHN/kernels/wpapsk_kernel.cl
Build log: In file included from /tmp/OCL7202T3.cl:13:
In file included from /home/user/JohnTheRipper/run/kernels/opencl_cmac.h:24:
/home/user/JohnTheRipper/run/kernels/opencl_aes.h:316:4: error: unsupported: typedef changes linkage of anonymous type, but linkage was already computed
        } aes_block_t;
          ^
/home/user/JohnTheRipper/run/kernels/opencl_aes.h:313:15: note: use a tag name here to establish linkage prior to definition
        typedef union {
                     ^
                      aes_block_t
1 error generated.

error: Clang front-end compilation failed!
Frontend phase failed compilation.
Error: Compiling CL to IR

Error -11 building kernel $JOHN/kernels/wpapsk_kernel.cl. DEVICE_INFO=522
OpenCL CL_BUILD_PROGRAM_FAILURE (-11) error in opencl_common.c:1177 - clBuildProgram failed.
```

--- a/src/opencl/sspr_kernel.cl
+++ b/src/opencl/sspr_kernel.cl
@@ -41,23 +41,23 @@
 #define SHA512_DIGEST_LENGTH 64
 #endif
 
-typedef union {
+typedef union out_u {
 	uchar b[BINARY_SIZE_MIN];
 	uint w[BINARY_SIZE_MIN / sizeof(uint)];
 	uint64_t W[BINARY_SIZE_MIN / sizeof(uint64_t)];
 } out_t;
 
-typedef union {
+typedef union hash_u {
 	uchar b[SHA_DIGEST_LENGTH];
 	uint w[SHA_DIGEST_LENGTH / sizeof(uint)];
 } hash_t;
 
-typedef union {
+typedef union hash256_u {
 	uchar b[SHA256_DIGEST_LENGTH];
 	uint w[SHA256_DIGEST_LENGTH / sizeof(uint)];
 } hash256_t;
 
-typedef union {
+typedef union hash512_u {
 	uchar b[SHA512_DIGEST_LENGTH];
 	uint w[SHA512_DIGEST_LENGTH / sizeof(uint)];
 	uint64_t W[SHA512_DIGEST_LENGTH / sizeof(uint64_t)];

--- a/src/opencl_aes.h
+++ b/src/opencl_aes.h
@@ -310,7 +310,7 @@ AES_ige_decrypt(AES_SRC_TYPE void *_in, AES_DST_TYPE void *_out,
 	AES_SRC_TYPE uchar *in = _in;
 	AES_DST_TYPE uchar *out = _out;
 
-	typedef union {
+	typedef union aes_block_u {
 		ulong data[N_WORDS];
 		uchar bytes[AES_BLOCK_SIZE];
 	} aes_block_t;

--- a/src/opencl_rawsha256.h
+++ b/src/opencl_rawsha256.h
@@ -44,7 +44,7 @@
 }
 
 //Data types.
-typedef union {
+typedef union buffer_32_u {
 	uint8_t mem_08[4];
 	uint16_t mem_16[2];
 	uint32_t mem_32[1];

--- a/src/opencl_rawsha512.h
+++ b/src/opencl_rawsha512.h
@@ -44,7 +44,7 @@
 }
 
 //Data types.
-typedef union {
+typedef union buffer_64_u {
 	uint8_t mem_08[8];
 	uint16_t mem_16[4];
 	uint32_t mem_32[2];

--- a/src/opencl_sha256crypt.h
+++ b/src/opencl_sha256crypt.h
@@ -30,7 +30,7 @@
 #define KEYS_PER_CORE_GPU       1
 
 //Data types.
-typedef union {
+typedef union buffer_32_u {
 	uint8_t mem_08[4];
 	uint16_t mem_16[2];
 	uint32_t mem_32[1];

--- a/src/opencl_sha512crypt.h
+++ b/src/opencl_sha512crypt.h
@@ -48,7 +48,7 @@
 #define KEYS_PER_CORE_GPU       1
 
 //Data types.
-typedef union {
+typedef union buffer_64_u {
 	uint8_t mem_08[8];
 	uint16_t mem_16[4];
 	uint32_t mem_32[2];

